packages/assets/scss/02-molecules/_emergency-alert.scss

```
@@ -3,14 +3,14 @@
 @use "sass:color" as *;
 
 .ma__emergency-alert {
-  background-color: mix(#fff, $c-warning,90%);
+  background-color: mix($c-white, $c-warning,90%);

```

```suggestion
  background-color: $c-highlight-lightest;
```

actually this is the same as $c-highlight-lightest, and the lightest colors are usually used as background color